omod/src/main/webapp/pages/runReport.gsp

```diff
@@ -174,7 +174,13 @@ ${ ui.includeFragment("appui", "messages", [ codes: [
                         ${ ui.includeFragment("uicommons", "field/location", [
                                 formFieldName: "parameterValues[" + it.name + "]",
                                 label: it.labelOrName,
-                                initialValue: it.defaultValue ?: sessionContext.sessionLocation
+                                initialValue: it.defaultValue ?: sessionContext.sessionLocation,
+                                classes: ["location-list"]
```

Comment:
What happens when you do not use the location-list class?

Reply:
Hi @dkayiwa 
If I don´t add the class, the drop box will break outside the intended screen.
This happens because the "max length" an encounter can have. 
![image](https://user-images.githubusercontent.com/70152448/120344851-df0cd880-c2f1-11eb-9298-c8612251ead9.png)

On another note, I change the class name to drop-down-list, since it may be use in future for other drop-downs.

Reply:
> If I don´t add the class, the drop box will break outside the intended screen.

Is this caused by adding text as a parameter type?